easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2017a.eb

```diff
@@ -14,7 +14,7 @@ checksums = ['43a2f9466702fb1db31df98ae6677f15']
 
 dependencies = [
     ('zlib', '1.2.11'),
-    ('Szip', '2.1'),
+    ('Szip', '2.1.1'),
```

Comment:
I only saw this older config when I got a merge conflict.  Since I had it based on 2.1.1, I thought I update it my way ;)